src/Admin/DonationAdmin.php

```diff
@@ -21,11 +23,8 @@ class DonationAdmin extends AbstractAdmin
     protected function configureShowFields(ShowMapper $show)
     {
         $show
-            ->add('isFinished', 'boolean', [
-                'label' => 'Don terminé ?',
-            ])
-            ->add('isSuccessful', 'boolean', [
-                'label' => 'Don réussi ?',
+            ->add('status', null, [
+                'label' => 'Status du don',
```

Comment:
1) Statut* (same for other labels)
2) which status it shows? `subscription_in_progress` or `waiting_confirmation`? maybe translate these values?